gcloud/bigtable/row.py

```diff
@@ -253,3 +324,21 @@ def to_pb(self):
         :returns: The converted current object.
         """
         return data_pb2.RowFilter(cells_per_column_limit_filter=self.num_cells)
+
+
+class StripValueTransformerFilter(_BoolFilter):
```

Comment:
Why not keep it with the other `_BoolFilter` derivatives?  Are you synching with an order in the protobuf spec, or the docs?